src/Suave/CookieSerialiser.fs

```diff
@@ -1,12 +1,11 @@
-﻿namespace Suave
+namespace Suave
 
 open System.IO
 
 type CookieSerialiser =
   abstract serialise : Map<string, obj> -> byte []
   abstract deserialise : byte [] -> Map<string, obj>
 
-#if NETSTANDARD1_5
 open System.Runtime.Serialization.Json
 
 type JsonFormatterSerialiser() =
```

Comment:
Do we really need ```JsonFormatterSerialiser``` type now? It is not used anywhere in the Suave code, and ```DataContractJsonSerializer``` cannot serialize ```FSharpMap```